phpunit.xml.dist

```diff
@@ -13,8 +13,10 @@
         <server name="SHELL_VERBOSITY" value="-1" />
         <!-- See: https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script -->
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
-        <!-- Set "SYMFONY_PHPUNIT_REMOVE" to "symfony/yaml" will install also "prophecy" see phpunit bridge docs for more information. -->
-        <server name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml"/>
+        <!-- Set "SYMFONY_PHPUNIT_VERSION" to use fixed phpunit version: https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script -->
+        <!-- <server name="SYMFONY_PHPUNIT_VERSION" value="8.5" /> -->
+        <!-- Set "SYMFONY_PHPUNIT_REMOVE" to "symfony/yaml" to install "prophecy": https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script -->
+        <!-- <server name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml"/> -->
```

Comment:
i think this is not set per default, right?

Reply:
@nnatter this was never part of symfony skeleton we activily decided to make prophecy always available as it does not hurt it does more hurt when not setting the SYMFONY_PHPUNIT_REMOVE parameter and so I would keep the parameter here.